sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/Models/TagDetails.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary> Tag details. </summary>
+    public partial class TagDetails
```

Comment:
Follow the naming convention of [Resource]Data

Reply:
Done